app/src/main/java/com/nilhcem/blenamebadge/ui/drawer/DrawerActivity.kt

```diff
@@ -217,7 +231,7 @@ class DrawerActivity : AppCompatActivity(), DrawerNavigator, NavigationView.OnNa
     }
 
     override fun saveImportFile(uri: Uri?) {
-        if (StorageUtils.copyFileToDirectory(this, uri)) Toast.makeText(this, R.string.success_import_json, Toast.LENGTH_SHORT).show()
```

Comment:
Is there any reason for removing the toast of successful import? Can't we keep both the toast and the updating of the list?

Reply:
We can, but if the item is successfully imported, it directly shows up in the list. Are you sure the user needs confirmation even when he can see that the file is shown in the list?

Reply:
I think we can show a toast. If the user has a lot of saved badges, the imported badge might not be immediately visible.

Reply:
@yashk2000 ok, just a min. let me bring it back

Reply:
@yashk2000 Check , Updated